continuous-deployment/readme.md

```diff
@@ -1,7 +1,14 @@
 # Continuous Deployment
```

Comment:
I would suggest trying to consolidate the content you put under `Pipelines` and here. Seems that the recipe is really trying to describe how to implement Continuous **Delivery** or **Deployment** (yes there is a distinct difference between these) across multiple environments using GitHub Actions. Pipelines term really only applies in AzDO and Jenkins AFAIK. So readers might find this confusing. The readers will more associate your content as a solution to CD rather than pipeline.

Reply:
Thats an interesting view on things @charleszipp. Will consolidate and update the two areas.